.github/ISSUE_TEMPLATE/bug-chinese.yml

```
@@ -1,75 +1,113 @@
+    validations:
+      required: true
+
   - type: textarea
-    id: reproduce-steps
+    id: reproduce
     attributes:
-      label: "复现方法"
-      placeholder: "复现该 Bug 的步骤："
+      label: 复现步骤
+      value: |
+        1. 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)